dev_guide/routes.adoc

```diff
@@ -133,10 +134,33 @@ custom certificates. Any non-SNI traffic received on port 443 is handled with
 TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
+endif::[]
 
 Further information on all types of
 xref:../architecture/core_concepts/routes.adoc#secured-routes[TLS termination]
 as well as
 xref:../architecture/core_concepts/routes.adoc#path-based-routes[path-based
 routing] are available in the
 xref:../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[Architecture section].
+
+ifdef::openshift-online[]
+[[custom-route-and-hosts-and-certificates-restrictions]]
+== Restrictions
+
+Currently, custom route hosts are not permitted. Instead, the following host
```

Comment:
I see what @Miciah is saying here. So something like "Custom route hosts are only available for users with paid accounts", except there's bound to be a better way to indicate a paid account without saying "paid accounts". In the older version of OpenShift Online, we had the "Bronze Plan" and "Silver Plan" for the paid tiers. Do we know yet what the paid tiers of the new Online will be called?